<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ingresar</class>
 <widget class="QDialog" name="Ingresar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="Vertices_LineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Numero_V_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero de Vertices:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Vertices_Submit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Continuar</string>
   </property>
  </widget>
  <widget class="QLabel" name="Aristas_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero de aristas:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Aristas_textEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Aristas_Submit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Continuar</string>
   </property>
  </widget>
  <widget class="QLabel" name="Ingresar_vertices_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ingresa Vertices:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Ingresar_V_submit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Continuar</string>
   </property>
  </widget>
  <widget class="QLabel" name="VOrigen_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>V. Origen:</string>
   </property>
  </widget>
  <widget class="QLabel" name="V_destino_label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>V. Destino:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="V_origen_textEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="V_destino_EditText">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Registrar_camino_submit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Registrar camino</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Go_to_solution">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Pasar a Soluciones del Grafo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Ingresa_V_LineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Vertices_Submit</sender>
   <signal>clicked()</signal>
   <receiver>Aristas_label</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>11</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Vertices_Submit</sender>
   <signal>clicked()</signal>
   <receiver>Aristas_textEdit</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Vertices_Submit</sender>
   <signal>clicked()</signal>
   <receiver>Aristas_Submit</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Aristas_Submit</sender>
   <signal>clicked()</signal>
   <receiver>Ingresar_vertices_label</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>24</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Aristas_Submit</sender>
   <signal>clicked()</signal>
   <receiver>Ingresar_V_submit</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Ingresar_V_submit</sender>
   <signal>clicked()</signal>
   <receiver>VOrigen_label</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Ingresar_V_submit</sender>
   <signal>clicked()</signal>
   <receiver>V_origen_textEdit</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>90</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Ingresar_V_submit</sender>
   <signal>clicked()</signal>
   <receiver>Ingresar</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Ingresar_V_submit</sender>
   <signal>clicked()</signal>
   <receiver>Registrar_camino_submit</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Ingresar_V_submit</sender>
   <signal>clicked()</signal>
   <receiver>V_destino_label</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Aristas_Submit</sender>
   <signal>clicked()</signal>
   <receiver>Ingresa_V_LineEdit</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Ingresar_V_submit</sender>
   <signal>clicked()</signal>
   <receiver>V_destino_EditText</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
